yasuke-server/src/services/token.service.ts

```diff
@@ -68,20 +78,37 @@ export class TokenService {
   }
 
   async listTokens(
```

Comment:
Can we leave listTokens method as is and create a new method, `listTokensWithAuction` (or something like that) for this change.

That means the controller will also have to expose another endpoint for this purpose. and the GUI guys will need to change the call to listTokensWithAuction whenever they need tokenInfo with auction info.

Reply:
Okay. I'll do this. thank you.